Python/libraries/datatypes-timex-expression/CreatePackage.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off
+echo *** Building Microsoft.Recognizers.Text.Timex
+setlocal
+setlocal enabledelayedexpansion
+setlocal enableextensions
+
+if not exist ..\dist mkdir ..\dist
+if exist ..\dist\Microsoft.Recognizers.Text.Timex*.whl erase /s ..\dist\Microsoft.Recognizers.Text.Timex*.whl
+python setup.py bdist_wheel -d ..\dist\
+
+set error=%errorlevel%
+set packageName=Microsoft.Recognizers.Text.Timex
```

Comment:
This is not the name matching nuget/npm nor following naming of the other packages. Shouldn't it be something like recognizers_text_data-types_timex-expression?
Nuget is Microsoft.Recognizers.Text.DataTypes.TimexExpression and npm is @microsoft/recognizers-text-data-types-timex-expression